week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,33 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
+  fillUp: function () {
+    return (this.volume = this.volume + 100);
     // calling this function should pour your bottle full (volume = 100);
   },
   pour: function () {
+    // if (this.volume <= 90) {
+    //   return (this.volume += 10);
+    // }
+    return this.volume <= 90 ? (this.volume += 10) : this.volume;
+
     // calling this function should increase your bottle volume by 10 unit;
   },
-  drink: function() {
+  drink: function () {
+    // if (this.volume >= 10) {
+    //   return (this.volume -= 10);
+    // }
+    return this.volume >= 10 ? (this.volume -= 10) : this.volume;
     // calling this function should decrease your bottle volume by 10 unit;
   },
   isFull: function () {
+    return this.volume >= 100 ? true : false;
     // this function should return true if your bottle is empty;

```

Comment:
if you have and if statement and inside of it you are returning true and false only, you don't need the (if statment )at all
another issue is the comparison using > greater sign, can you have a volume over a hundred ?